external-adapter/src/validator.ts

```diff
@@ -6,13 +6,15 @@ import { AdapterErrorResponse } from '@chainlink/types'
 export class Validator {
   input: any
   customParams: any
+  options: Record<string, any[]>
   validated: any
   error: AdapterError | undefined
   errored: AdapterErrorResponse | undefined
 
-  constructor(input = {}, customParams = {}) {
+  constructor(input = {}, customParams = {}, options = {}) {
```

Comment:
Adding an additional parameter to our Validator that validates options.

So if I had the input
```
{
id: 1
data: {
  coin: "BTC"
}
}
```
and want to make sure that `coin` = `BTC` or `ETH` I could pass
```
{
 coin: ["BTC", "ETH"]
}
```